File: src/main/scala/resonantinduction/core/resource/ItemHandCrank.java
package resonantinduction.core.resource;

import net.minecraft.item.Item;
import net.minecraft.world.World;

public class ItemHandCrank extends Item
{
	public ItemHandCrank(int id)
	{
		super(id);
		setMaxStackSize(1);
	}

	@Override
	public boolean shouldPassSneakingClickToBlock(World world, int x, int y, int z)
	{
		return true;
	}
}

File: src/main/scala/resonantinduction/mechanical/gear/ItemHandCrank.scala
package resonantinduction.mechanical.gear

import net.minecraft.item.Item
import net.minecraft.world.World

class ItemHandCrank extends Item
{
  setMaxStackSize(1)
  
  def shouldPassSneakingClickToBlock(world: World, x: Int, y: Int, z: Int): Boolean =
  {
    return true
  }
}